tests/ui/transmutability/structs/repr/unhandled_error_ice_117491.rs

```diff
@@ -0,0 +1,23 @@
+//~ ERROR: cycle detected
```

Comment:
Why does the test name not mention "cycle" at all 😆  -- the fact that it's "unhandled" is the least useful part of the error description, imo

Please change that

Reply:
😆 yea naming stuff is hard. I also left a comment explaining the test